docker-build/action.yml

```diff
@@ -1,38 +1,46 @@
 name: docker-build
-description: Build a Mozcloud service image
+description: >-
+  Build a Docker container image, providing standard MozCloud tags for supported registries.
+  Outputs a newline-delimited list of images in `image_tags` to be consumed by the 
+  [docker-push](../docker-push/README.md) action.
 inputs:
   image_name:
     description: Name to give to the built image
+    example: my-service
```

Comment:
As mentioned in the description, these `example` fields are custom and required to use the script.


Though my editor complains

<img width="526" height="111" alt="Image" src="https://github.com/user-attachments/assets/9aa2f342-12b6-4a0d-b5d6-61f093114ac5" />


I think it's safe to use them. I noticed we specify a `type` in another action, which Github doesn't seem to mind when consuming the `action.yml` for that action

<img width="499" height="120" alt="Image" src="https://github.com/user-attachments/assets/aec560ef-a4b2-44b1-8ff6-4d343e7634f4" />